homeassistant/components/usb/__init__.py

```diff
@@ -125,11 +126,11 @@ async def async_setup(hass: HomeAssistant, config: ConfigType) -> bool:
     return True
 
 
-def _fnmatch_lower(name: str | None, pattern: str) -> bool:
-    """Match a lowercase version of the name."""
+def _fnmatch_case_insensitive(name: str | None, pattern: str) -> bool:
+    """Match a name case-insensitively."""
     if name is None:
         return False
-    return fnmatch.fnmatch(name.lower(), pattern)
```

Comment:
Well, this would have worked fine on Windows 😁 (since it uses `os.path.normcase()` under the hood).